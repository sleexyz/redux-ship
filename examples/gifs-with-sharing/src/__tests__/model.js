// @flow
import * as Test from '../test';
import * as Model from '../model';

Test.snapshotReduce(Model.reduce, {
  counter: {
    patch: {counter: {type: 'IncrementByOne'}},
    state: Model.initialState,
  },
  randomGifs: {
    patch: {randomGifs: {
      patch: {type: 'LoadStart'},
      tag: Object.keys(Model.initialState.randomGifs)[0],
    }},
    state: Model.initialState,
  },
});
