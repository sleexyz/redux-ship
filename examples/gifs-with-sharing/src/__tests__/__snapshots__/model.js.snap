exports[`test counter 1`] = `
Object {
  "counter": Object {
    "count": 1
  },
  "randomGifs": Object {
    "cat": Object {
      "gifUrl": null,
      "isLoading": false
    },
    "dog": Object {
      "gifUrl": null,
      "isLoading": false
    },
    "gify": Object {
      "gifUrl": null,
      "isLoading": false
    },
    "minion": Object {
      "gifUrl": null,
      "isLoading": false
    }
  }
}
`;

exports[`test randomGifs 1`] = `
Object {
  "counter": Object {
    "count": 0
  },
  "randomGifs": Object {
    "cat": Object {
      "gifUrl": null,
      "isLoading": true
    },
    "dog": Object {
      "gifUrl": null,
      "isLoading": false
    },
    "gify": Object {
      "gifUrl": null,
      "isLoading": false
    },
    "minion": Object {
      "gifUrl": null,
      "isLoading": false
    }
  }
}
`;
